libs/core/langchain_core/output_parsers/json.py

```diff
@@ -221,7 +221,7 @@ def get_format_instructions(self) -> str:
         if self.pydantic_object is None:
             return "Return a JSON object."
         else:
-            schema = self.pydantic_object.schema()
+            schema = {k: v for k, v in self.pydantic_object.schema().items()}
```

Comment:
a comment here would be great! otherwise looks good

Reply:
@hwchase17  
I've applied the changes, thank you :)

Reply:
I have also made the same changes in `/langchain/langchain/output_parsers/yaml.py` and `/langchain/langchain/output_parsers/pydantic.py`.